docs/modules/deployment-launcher/manage-deployments.md

```diff
@@ -0,0 +1,98 @@
+<%( TOC )%>
+
+# Manage deployments
+
+Open the Deployment Launcher window by selecting **SpatialOS** > **Deployment Launcher** from the Unity Editor menu.
+
+## UI walkthrough
+
+The section of the Deployment Launcher window that you use to manage live deployments can be found under the **Live Deployments** label.
+
+Initially, the Deployment Launcher does not know of any deployments. It will only update its known list of deployments when requested.
+
+<img src="{{assetRoot}}assets/modules/deployment-launcher/manage-deployments-empty.png" style="margin: 0 auto; width: auto; display: block;" />
+
+<%(#Expandable title="Buttons")%>
+
+| Button name | Description |
+| --- | --- |
+| Refresh | When pressed, the Deployment Launcher retrieves a list of cloud deployments currently running in your SpatialOS project. |
+
+<%(/Expandable)%>
+
+### Listed deployments
+
+When there are live cloud deployments running, this section is updated with a list of these live deployments and some metadata about them. The deployment metadata is **read-only**.
+
+<img src="{{assetRoot}}assets/modules/deployment-launcher/manage-deployments.png" style="margin: 0 auto; width: auto; display: block;" />
+
+<%(#Expandable title="Metadata")%>
+
+| Metadata | Example |
+| --- | --- |
+| Deployment name | The name of the given cloud deployment. For example, `testmobile13052019`. |
```

Comment:
just leaving a comment here that I still don't like `testmobile13052019` as an example xD